sections/introduction.tex

```diff
@@ -1,69 +1,109 @@
 \section{Introduction}
 
-Over the last several years, there has been a growing interest in learning
-outside of the traditional classroom setting. Whether it is because departments
-do not offer classes in a certain area, or because the format of semester-long
-courses does not mesh well with the topics covered, there is a demand for new
-approaches to teaching the skills and ideas needed to do science. This is
-particularly true in teaching computational and data analytic techniques,
+Science has recently seen a large growth in the use of
+computationally intensive and data-centric methods. Researchers are
+increasingly using programming languages such as R or Python, and utilizing
+complex algorithms in applied statistics and machine learning in order to
+perform their research \cite{momcheva2015astro}.
+
+With an increased focus on computational methods
+comes new challenges in teaching these techniques, and new approaches
+towards sharing knowledge with fellow scientists.
+A rapidly growing approach to scientific training is learning
+outside of the traditional semester-long classroom setting.
+This is particularly true
+for teaching computational and data analytic techniques,
 as instructors must teach both conceptual and methodological information
-at the same time. One common way to teach this type of skills is a short,
-time-bounded learning event, such as a bootcamp or short-course
-\cite{wilson2016software}. These attempt to compress several
+simultaneously. A common way to teach these skills is a short,
+time-bounded learning event, such as a bootcamp or workshop
+\cite{wilson2016software}. These  courses attempt to compress several
 topics into an intensive learning session that is usually held over one or
 several days.
 
-Workshops often follow the same formula, roughly described here: First,
-instructors develop materials on their own computers, sharing them with
+As these new learning models are adopted, it opens
+opportunities for developing new technology and models for pedagogy that are
+focused on "hands-on" learning. Teaching a class in which students
+must simultaneously learn both conceptual and methodological information
+(usually with a strong hands-on component on personal laptops) poses
+many challenges for the standard "bootcamp" model of
+learning. Here we describe
+recent work in utilizing cloud-based infrastructure to enhance this learning
+experience, and to streamline the ability of instructors to
+teach material that focuses on data analytic techniques. Our approach
+utilizes recent advances in cloud- and cluster- based technology, such
+as Kubernetes, Container technology, and the Jupyter framework for computing.
```

Comment:
isn't Kubernetes a Container technology?

So, maybe Container technology (e.g. Kubernetes, Docker) and the Jupyter framework for computing.